### PR TITLE
Dev-Typescript

### PR DESCRIPTION
**Pull request by PikaJoyce**
Final PR to merge into develop. There are no new additions to the previous PR.

**Contributing to Twilio**

> All third-party contributors acknowledge that any contributions they provide will be made under the same open-source license that the open-source project is provided under.

- [x] I acknowledge that all my contributions will be made under the project's license.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -83,12 +91,23 @@ New Features
 2.9.0 (December 2, 2020)
 ========================
 
+New Features
```
Comment (makarandp0) [inline CHANGELOG.md:94]: delete this duplicate entry.

File: CHANGELOG.md [hunk 2]
```diff
@@ -83,12 +91,23 @@ New Features
...
+
+2.8.0 (in progress)
+===================
+
 Changes
 -------
 
 - Previously, `Room.isRecording` indicated whether recording is enabled for the Room.
 Now it indicates if the Track published to the Room are being recorded. If recording is
-enabled for the Room, then `Room.isRecording` is set to `true` when the first Track is published
```
Comment (makarandp0) [inline CHANGELOG.md:91]: please undo this change.

**Review by makarandp0 — CHANGES_REQUESTED** [review]
looks like we have some bad merge in CHANGELOG.md, Please fix it, but otherwise it looks good. We can work on real text for changelog just before release.

File: CHANGELOG.md [hunk 3]
```diff
@@ -2,6 +2,14 @@ The Twilio Programmable Video SDKs use [Semantic Versioning](http://www.semver.o
 
 **Support for 1.x will cease on December 4th, 2020**. This branch will only receive fixes for critical issues until that date. Check [this guide](https://www.twilio.com/docs/video/migrating-1x-2x) when planning your migration to 2.x. For details on the 1.x changes, go [here](https://github.com/twilio/twilio-video.js/blob/support-1.x/CHANGELOG.md).
 
+2.11.0 (In Progress)
+====================
+
+New Features
+------------
+
+- twilio-video now includes TypeScript definitions
```
Comment (manjeshbhargav) [inline CHANGELOG.md:11]:
```suggestion
- You can now import type definitions for the SDK APIs to your project as shown below. (JSDK-3007)

  ` ` `ts
    import * as Video from 'twilio-video';
  ` ` `
```

File: README.md [hunk 4]
```diff
@@ -56,6 +56,28 @@ Using this method, you can `require` twilio-video.js like so:
 const Video = require('twilio-video');
 ```
 
+TypeScript definitions can now be imported using this method.
```
Comment (manjeshbhargav) [inline README.md:59]:
```suggestion
TypeScript definitions can now be imported as shown below.
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -81,7 +93,7 @@ New Features
   ```
 
 2.9.0 (December 2, 2020)
-========================
+===================
```
Comment (makarandp0) [inline CHANGELOG.md:96]:
```suggestion
========================
```

**Review by makarandp0 — APPROVED** [review]

**Review by manjeshbhargav — APPROVED** [review]
